app/components/primer/tooltip_component.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Primer
+  # The Tooltip component is a wrapper component that will apply a tooltip
+  class TooltipComponent < Primer::Component
+    DIRECTION_DEFAULT = ""
+    ALIGNMENT_DEFAULT = ""
+    MULTILINE_DEFAULT = false
+    DELAY_DEFAULT = true
+
+    DIRECTION_OPTIONS = [DIRECTION_DEFAULT] + %w(
```

Comment:
why strings and not symbols?

Reply:
It was easier to copy / paste ;) Happy to convert to strings if we'd like.

Reply:
I'd vote for symbols as well ❤️